pyproject.toml

```diff
@@ -331,7 +331,7 @@ types-setuptools = "^67.2.0"
 memory-profiler = "^0.61.0"
 psutil = "^5.8.0"
 mypy-extensions = "^0.4.3"
-sanic-testing = ">=21.12.0,<22.9.0"
+sanic-testing = ">=21.12.0,<23.4.0"
```

Comment:
There are a lots of tests failing I think because of this upgrade, please check that sanic-testing is compatible with the version of sanic we have pinned, if not don't try to upgrade, the major.minor version need to be aligned for these dependencies.

Reply:
Also as reference for the future, if you upgrade a dependency and it breaks tests, as part of the PR you will have to fix these breaking changes and adapt code.